lib/selbot2/wiki.rb

```diff
@@ -12,19 +12,29 @@ def execute(message, query)
       replies = replies_for("pagename:#{query}")
 
       if replies.empty?
-        replies = replies_for(query)
-      end
-
-      if replies.empty?
-        message.reply "No results."
-        return
+        google_fallback(message, query)
       end
 
       replies.each_with_index do |resp, idx|
         message.reply "#{idx + 1}: #{resp}"
       end
     end
 
+    def google_fallback(message, query)
+      puts "Falling back to google search"
+      query += " site:github.com/seleniumhq/selenium/wiki"
+      resp   = JSON.parse(RestClient.get("https://www.googleapis.com/customsearch/v1?cx=005991058577830013072%3Awcdcytdwbcy&key=AIzaSyC3Nf0aBxyTLp9aZZkbAJkq0sXXWU35bJ4&num=1&q=#{URI.escape query}"))
+      result = resp.fetch('items').first
```

Comment:
If the search doesn't have any results, this `fetch` will throw an error, may be better to do:

```
result = resp['items']  # will return nil if item isn't found

if result.nil?
  message.reply "No results."
else
  item = result.first
  message.reply "#{item['title']}: #{item['link']}"
end